pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -89,6 +89,88 @@ func getApplicablePodNameForAppFramework(cr splcommon.MetaObject, ordinalIdx int
 	return fmt.Sprintf("splunk-%s-%s-%d", cr.GetName(), podType, ordinalIdx)
 }
 
+// runCustomCommandOnSplunkPods creates the required directory for the pod/s
```

Comment:
nit: We can generalize, instead of mentioning it as directory creation, as that is what we are doing in this.